README.md

```diff
@@ -1,2 +1,27 @@
-# .github
-Readme information for TheCookieMC
+## Welcome to TheCookieMC!
+
+![CookieMC banner](https://raw.githubusercontent.com/TheCookieMC/BugReports/main/cookiemc.png?token=GHSAT0AAAAAABZTNNIHRBVS66ROEBB6OHVSY3O5CUA)
+
+Hello and Welcome to the offical github page for TheCookieMC! Here, you will find all of our API's information and documentation!
+
+### What is TheCookieMC?
+
+TheCookieMC is a Java minigame network!
+
+We started development in 2019, and after some discussion, we stopped development due to developers not being available and Minecraft declining in players.
+
+In 2021, we saw the quality of other Java servers decline: we had the idea to continue with development, redoing old plugins and making new ones to finish our idea and create a fun server.
+
+Our server is currently in a closed beta state, we'll sometimes post invites on our discord server to help us test & we may eventually change to a fully fledged server!
+
+### Important Links
+
+IP Address: | play.thecookiemc.net
+<br>Discord Link: | https://discord.gg/TSf6rgx4MG
+<br>Punishment Appeals: | http://thecookiemc.net/appeal
+<br>Twitter: | https://twitter.com/PlayCookieMC
+      
+
+### One last note
+
+Thanks for checking out our page on GitHub! At this time, we do not have any plans to publish any of our work. The only thing that is currently avalible on our github page is information about our [API](https://github.com/TheCookieMC/API). If you have any questions, you can find more information from staff members in our offical [discord server](https://discord.gg/TSf6rgx4MG). Thanks for stopping by :)
```

Comment:
Half of this doesn't need to be specified, an api link and some description that this is where our documentation is, is fine.